spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/endpoints.adoc

```diff
@@ -459,49 +483,53 @@ Finally, if you need access to web-framework-specific functionality, you can imp
 
 [[actuator.endpoints.implementing-custom.input]]
 ==== Receiving Input
-Operations on an endpoint receive input via their parameters.
-When exposed via the web, the values for these parameters are taken from the URL's query parameters and from the JSON request body.
-When exposed via JMX, the parameters are mapped to the parameters of the MBean's operations.
+Operations on an endpoint receive input through their parameters.
+When exposed over the web, the values for these parameters are taken from the URL's query parameters and from the JSON request body.
+When exposed over JMX, the parameters are mapped to the parameters of the MBean's operations.
 Parameters are required by default.
 They can be made optional by annotating them with either `@javax.annotation.Nullable` or `@org.springframework.lang.Nullable`.
 
-Each root property in the JSON request body can be mapped to a parameter of the endpoint.
+You can map each root property in the JSON request body to a parameter of the endpoint.
 Consider the following JSON request body:
 
+====
 [source,json,indent=0,subs="verbatim"]
 ----
 	{
 		"name": "test",
 		"counter": 42
 	}
 ----
+====
 
-This can be used to invoke a write operation that takes `String name` and `int counter` parameters, as shown in the following example:
+You can use this to invoke a write operation that takes `String name` and `int counter` parameters, as the following example shows:
```

Comment:
The only rule that I try to enforce is to have some kind of lead-in to the code (or image or whatever). I've switched to this wording because it is a bit shorter, but it's OK to have both (though better to have just the one). In other words, it's not official. Rather, it's what I'm using at present (again, because it's a smidgeon shorter).

Reply:
Can we please try to avoid this sort of churn in the documentation? It makes consistency across all of the docs a moving target, and one that I suspect we'll never reach as a result.

Reply:
I have yet to come up with a lead-in to code that I think is really excellent. I changed it to "as the following example shows" because it's shorter and more direct (ditching the preposition). Honestly, though, if I think of or find something better in the future, I'll change it again, because it will be better.